azure-transfer-wf/main.nf

```diff
@@ -153,7 +161,7 @@ workflow AzureTransferWf {
     // example k8s emptyDir: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
     if (params.cleanup && params.local_dir == 'NO_DIR') {
       cleanup(
-        DownloadData.out.data_file.concat(DownloadMeta.out.payload_json).collect(),
```

Comment:
this will leave the downloaded metadata json (if it's been downloaded) not be able to be cleaned up.